src/GSMModem.h

```diff
@@ -37,6 +37,7 @@ class GSMModem {
       @return modem IMEI number
    */
   String getIMEI();
+  String getICCID();
```

Comment:
If you can add a comment like the `getIMEI()` one above, that would be great.

Reply:
done and done :)
thank you, @sandeepmistry